ethereum/WebaverseERC1155.sol

```diff
@@ -91,34 +91,36 @@ contract WebaverseERC1155 is
 
     /**
      * @return Returns the token URI against a particular token id.
-     * e.g. [baseURI]/[_id]
+     * e.g. https://tokens.webaverse.com/1
      */
     function uri(uint256 _id) public view override returns (string memory) {
-        string memory _baseURI = baseURI();
-        string memory _cid = Strings.toString(_id);
-        string memory _uri;
-        if (bytes(_baseURI).length > 0) {
-            _uri = string(abi.encodePacked(_baseURI, "/", _cid));
-        } else {
-            _uri = _cid;
-        }
-        return _uri;
+        string memory baseURI = _baseURI();
+        return bytes(baseURI).length != 0 ? string(abi.encodePacked(baseURI, _id.toString())) : '';
     }
 
     /**
-     * @dev Set token uri
-     * @param tokenId Token id to set the uri to
-     * @param _uri The uri to set for the token
+     * @dev get token contentURL
+     * @param tokenId Token id to set the ContentURL to
+     * @param _uri The contentURL to set for the token
      */
-    /// _uri === [cid] or https://[linktofile]
-    function setTokenURI(uint256 tokenId, string memory _uri)
+    function setTokenContentURL(uint256 tokenId, string memory _uri)
         public
         onlyMinter
     {
         require(bytes(_uri).length > 0, "ERC1155: URI must not be empty");
         _tokenURIs[tokenId] = _uri;
     }
 
+    /**
+     * @dev get token ContentURL
```

Comment:
`contentUrl` as it's variable named, not with first capital

Reply:
fixed it